src/realm/exec/CMakeLists.txt

```diff
@@ -31,15 +29,20 @@ set_target_properties(RealmBrowser PROPERTIES
 )
 target_link_libraries(RealmBrowser Storage)
 
-add_executable(Realm2JSON EXCLUDE_FROM_ALL realm2json.cpp )
```

Comment:
It should still be `EXCLUDE_FROM_ALL` like the other executables, I think. Otherwise you'd always end up building it when developing locally.